### DISCUSSION TITLE
How to integrate echarts/D3Js charts into Apache Superset :

### DISCUSSION THREAD

**Discussion by GaryMam** [apache/superset#15979] [Q&A / Help]
Hi All, 
I am working on one POC of how charts are integrated in apache superset and trying to add/delete new/existing charts into SupertSet. Does anyone know if I can delete existing chart from superset and then add it back?If yes then how? I tried to delete one chart “World Map” from my local which was present in the locations as mentioned below but I still see world map in my superset which is 
**http://localhost:8080/superset/welcome/**

Location I removed world map files from are as below :
1.Users/gaurangmamgain/Documents/GitHub/superset/superset-frontend/node_modules/@superset-ui

2. /Users/gaurangmamgain/Documents/GitHub/superset/superset-frontend/src/visualizations/presets. - removed from Mainpreset.js file

3. /Users/gaurangmamgain/Documents/GitHub/superset/superset-frontend/node_modules/@superset-ui/legacy-plugin-chart-world-map/esm

**Comment by spartyhacker** [comment]
Same here. But I think this might a rookie mistake, like not knowing how to start the server etc..
